src/schema/objects/metadata.yaml

```diff
@@ -744,7 +744,7 @@ EffectiveEchoSpacing:
     between lines in the phase-encoding direction,
     defined based on the size of the reconstructed image in the phase direction.
     It is frequently, but incorrectly, referred to as "dwell time"
-    (see `DwellTime` parameter below for actual dwell time).
+    (see `"DwellTime"` parameter for actual dwell time).
```

Comment:
removed the `below` because this might not make sense in all context if this description was used in a table where `DwellTime` does not appear